src/box/lua/slab.cc

```diff
@@ -134,38 +135,35 @@ lbox_slab_stats(struct lua_State *L)
 static int
 lbox_slab_info(struct lua_State *L)
 {
-	struct memtx_engine *memtx;
-	memtx = (struct memtx_engine *)engine_by_name("memtx");
-
-	struct allocator_stats stats;
-	memset(&stats, 0, sizeof(stats));
+	uint64_t items_size = box_slab_info(BOX_SLAB_INFO_ITEMS_SIZE);
+	uint64_t items_used = box_slab_info(BOX_SLAB_INFO_ITEMS_USED);
+	uint64_t arena_size = box_slab_info(BOX_SLAB_INFO_ARENA_SIZE);
+	uint64_t arena_used = box_slab_info(BOX_SLAB_INFO_ARENA_USED);
+	uint64_t quota_size = box_slab_info(BOX_SLAB_INFO_QUOTA_SIZE);
+	uint64_t quota_used = box_slab_info(BOX_SLAB_INFO_QUOTA_USED);
 
 	/*
 	 * List all slabs used for tuples and slabs used for
 	 * indexes, with their stats.
 	 */
```

Comment:
Apparently this comment was copied from `lbox_slab_stats` long time ago. It doesn't make any sense here. Let's remove it while we are here.

Reply:
Fix it